### PR TITLE
Tickets/dm 11893

### PR DESCRIPTION
**Pull request by fjammes**

### CODE REVIEW

File: S15/tests/env-kube.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+# Parallel invokes the shell indicated by the SHELL environment variable
+export SHELL=$(type -p bash)
+
+function mysql_query {
+    sql="$1"
+    sleep_delay="$2"
+
+    SSH_CFG="$HOME/.lsst/qserv-cluster/ssh_config"
+	# ssh option for using configuration file
```
Comment (andy-slac) [inline S15/tests/env-kube.sh]: TAB

File: S15/tests/env-kube.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    user="qsmaster"
+    db="LSST"
+    port=4040
+
+    query_cmd="start=\$(date +%s.%N) && \
+        mysql -N -B --host \"127.0.0.1\" --port $port --user=$user $db -e \"$sql\" && \
+        end=\$(date +%s.%N) && \
+        echo \"Execution time: \$(python -c \"print(\${end} - \${start})\") sec\""
+
```
Comment (andy-slac) [inline S15/tests/env-kube.sh:31]: trailing spaces

File: S15/tests/dbbench/env.sh [hunk 3]
```diff
@@ -1,2 +1,2 @@
+export GOPATH=$HOME/go
```
Comment (andy-slac) [inline S15/tests/dbbench/env.sh:1]: Do you really need `go` for these tests?

File: S15/tests/env-kube.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
+if [ -z "$CLIENT_NODE" ]; then
+    echo "ERROR: undefined \$CLIENT_NODE"
+    exit 1
+fi
```
Comment (andy-slac) [inline S15/tests/env-kube.sh:6]: `node=${CLIENT_NODE:?}` does the same but is three lines shorter 😏

**Review by andy-slac — APPROVED** [review]
Looks OK

File: S15/tests/env-kube.sh (hunk 4 above)
Comment (fjammes) [reply S15/tests/env-kube.sh:6]: 👍

File: S15/tests/env-kube.sh (hunk 1 above)
Comment (fjammes) [reply S15/tests/env-kube.sh]: 👍

File: S15/tests/env-kube.sh (hunk 2 above)
Comment (fjammes) [reply S15/tests/env-kube.sh:31]: 👍

File: S15/tests/dbbench/env.sh (hunk 3 above)
Comment (fjammes) [reply S15/tests/dbbench/env.sh:1]: No

Comment (fjammes) [reply S15/tests/dbbench/env.sh:1]: But I leave it for now because I use `go env GOPATH` below...

Comment (andy-slac) [reply S15/tests/dbbench/env.sh:1]: Not sure I understand the logic. You only use `go` to set `$PATH`?

**Pull request merged**